### DISCUSSION TITLE
[一般解析区画]GPUノードへのジョブ指定につきまして

### DISCUSSION THREAD

**Discussion by szktkyk** [nig-sc/nigsc_homepage2#366] [Q&A] [answered]
下記のようなコマンドで、2023/9/21にジョブを投入しました。gfreeで確認するとノードが空いているようなのですが、なかなかジョブが入らない状況が続いています（現在2023/9/28）。
ジョブ投入のスクリプトに記載する設定に不備があるかどうか、ご確認いただけますでしょうか？
よろしくお願いいたします。

`qsub -l gpu -l cuda=4 test_dnabert2_gue.sh `

```sh
#!/bin/bash

#$ -cwd
#$ -V 
#$ -l gpu
#$ -l d_rt=03:00:00
#$ -l s_rt=03:00:00
#$ -l s_vmem=8G 
#$ -l mem_req=8G
#$ -N dnabert2_GUE_20230915
#$ -S /bin/bash

nvidia-smi
conda activate dna
export OPENBLAS_NUM_THREADS=1
cd /home/m216692/DNABERT_2/finetune/
sh scripts/run_dnabert2.sh /home/m216692/DNABERT_2/src/GUE 
sh scripts/run_dnabert1.sh /home/m216692/DNABERT_2/src/GUE 4
sh scripts/run_nt.sh /home/m216692/DNABERT_2/src/GUE 0
```

**Comment by sc-fs** [comment, marked as answer]
提示して頂いた、スクリプト及び、投入オプションには問題は見られません。

実行に時間が掛かる原因ですが、
まず、gfreeで表示されている内容は、現在の状況となります。
この内igt004は、有料利用のノードになりこちらには投入されません。
結果、gfreeの結果でも、空きが無い状況になっています。

また、gpu.qの待ち状態ですが、szktkykさんのＪＯＢを含め179JOBが待っている状況となります。
（状況については　https://sc.ddbj.nig.ac.jp/operation/）

ご迷惑をお掛けしますが、もうしばらくお待ちください。
